cloud/private-endpoints.mdx

```diff
@@ -67,6 +67,59 @@ Set up an AWS VPC endpoint to securely access Turso databases through AWS Privat
       </Step>
     </Steps>
 
+  </Tab>
+  <Tab title="us-east-2">
+
+    <Steps>
+      <Step title="Create VPC Endpoint">
+        Navigate to **VPC Dashboard** → **Endpoints** → **Create endpoint** and configure:
+
+        <AccordionGroup>
+        <Accordion title="Basic Settings">
+            - **Name**: Enter a descriptive name (e.g., `turso-database-endpoint`)
+            - **Service category**: Select "Other endpoint services"
+            - **Service name**: Enter `com.amazonaws.vpce.us-east-2.vpce-svc-0bd615901070ec214`
+            - Click **Verify service**
+        </Accordion>
+        <Accordion title="Network Configuration">
+            - Select your **VPC**
+            - Enable **DNS name**
+            - Choose **IPv4** for DNS record type
+            - Select subnets in supported AZs (`use2-az1` and/or `use2-az2`)
+            - Configure security groups and click **Create endpoint**
+        </Accordion>
+        </AccordionGroup>
+
+      </Step>
+      <Step title="Configure Security Groups">
+        Update security groups to allow proper communication:
+
+        **VPC Endpoint Security Group:**
+        - Inbound: Allow HTTPS (port 443) from your application security groups
+
+        **Application Security Groups:**
+        - Outbound: Allow HTTPS (port 443) to the VPC endpoint security group
+      </Step>
+      <Step title="Update Application Connections">
+          Replace public Turso endpoints with the new VPC endpoint URL format:
```

Comment:
It's not necessary "replacement" because user can already use `https://<database-name>.aws-us-east-2.turso.io` url, right? 

So we just need to say that you need to check that _regional_ url is used, no?